includes/class-amp-story-post-type.php

```diff
@@ -282,9 +335,46 @@ public static function filter_block_editor_settings( $editor_settings, $post ) {
 		if ( self::POST_TYPE_SLUG === $post->post_type ) {
 			unset( $editor_settings['colors'] );
 		}
+
+		if ( get_current_screen()->is_block_editor && isset( $editor_settings['styles'] ) ) {
+			foreach ( $editor_settings['styles'] as $key => $style ) {
+
+				// If the baseURL is not set or if the URL doesn't include theme styles, move to next.
+				if ( ! isset( $style['baseURL'] ) || ! self::is_theme_stylesheet( $style['baseURL'] ) ) {
+					continue;
+				}
+
+				/**
+				 * Filters the editor style to allow whitelisting it for AMP Stories editor.
+				 *
+				 * @param bool   $whitelisted If to whitelist the stylesheet.
+				 * @param string $base_url    The URL for the stylesheet.
+				 */
+				if ( false === apply_filters( 'amp_stories_whitelist_editor_style', false, $style['baseURL'] ) ) {
+					unset( $editor_settings['styles'][ $key ] );
+				}
+			}
+		}
 		return $editor_settings;
 	}
 
+	/**
+	 * Checks if a stylesheet is from the theme or parent theme.
+	 *
+	 * @param string $url Stylesheet URL.
+	 * @return bool If the stylesheet comes from the theme.
+	 */
+	public static function is_theme_stylesheet( $url ) {
+
+		if (
+			false !== strpos( $url, get_stylesheet_directory_uri() ) ||
+			false !== strpos( $url, get_template_directory_uri() )
+		) {
+			return true;
+		}
+		return false;
```

Comment:
Minor style point, but this function could be refactored to be just:

```
return (
    0 === strpos( $url, get_stylesheet_directory_uri() )
    ||
    0 === strpos( $url, get_template_directory_uri() )
);
```

Note also that testing for `0` is better than `false` because it needs to make sure that the string is at the beginning, not somewhere later (like as a query param).

Reply:
Sure, I'll make that change and then merge.